packages/frontend2/src/app/layout.tsx

```diff
@@ -7,7 +7,7 @@ import { env } from '~/env'
 import { TRPCReactProvider } from '~/trpc/react'
 import { restoreCollapsibleNavStateScript } from './_components/nav/consts'
 
-import './globals.css'
+import './styles/globals.css'
```

Comment:
Let's either make it _styles or move it outside app directory.

Reply:
got there by mistake